test_app/CMakeLists.txt

```diff
@@ -2,10 +2,10 @@
 # CMakeLists in this exact order for cmake to work correctly
 cmake_minimum_required(VERSION 3.5)
 
-set(EXTRA_COMPONENT_DIRS ../libsodium ../expat)
+set(EXTRA_COMPONENT_DIRS ../libsodium ../expat ../esp_encrypted_img)
```

Comment:
This we can avoid for now to avoid CI failures with v4.3 release. We will have to see on how to make build configurable based on IDF version that component supports.